WordPressKitTests/Utilities/LoggingTests.m

```diff
@@ -57,6 +57,17 @@ - (void)testLogging
     XCTAssertEqualObjects(self.logger.errorLogs, (@[@"This is an error log", @"This is an error log with an argument"]));
 }
 
+- (void)testLogging2
+{
+    WPKitLogError(@"This is an error log");
+    WPKitLogError(@"This is an error log %@", @"with an argument");
+    XCTAssertEqualObjects(self.logger.errorLogs, (@[@"This is an error log", @"This is an error log with an argument"]));
+
+    WPKitLogInfo(@"This is an info log");
+    WPKitLogInfo(@"This is an info log %@", @"with an argument");
+    XCTAssertEqualObjects(self.logger.infoLogs, (@[@"This is an info log", @"This is an info log with an argument"]));
+}
+
```

Comment:
This was introduced as part of [651b9e9b2aa269359511b0cbf96d54de1001d370](https://github.com/wordpress-mobile/WordPressKit-iOS/commit/651b9e9b2aa269359511b0cbf96d54de1001d370) which hints at debugging attempts. 

I guess we can remove it now?

```suggestion
```

Not that it _hurts_, but if we want to add some redundancy to the tests, it would help to spell it out clearly in the test name.

Reply:
Good catch. I should've deleted them 😄